code/modules/clothing/accessories/holster.dm

```diff
@@ -130,3 +142,49 @@
 	name = "hip holster"
 	desc = "A handgun holster slung low on the hip, draw pardner!"
 	icon_state = "holster_hip"
+
+//Sword holsters//
+
+/obj/item/clothing/accessory/holster/saber
+	name = "scabbard"
+	desc = "A sturdy brown leather scabbard with a gold trim, made to house a variety of swords."
+	icon_state = "sheath"
+	overlay_state = "sword"
+	slot = "utility"
+	can_hold = list(
+		/obj/item/tool/sword/nt/shortsword,
+		/obj/item/tool/sword/nt/longsword,
+		/obj/item/tool/sword/nt/scourge,
+		/obj/item/tool/sword/nt_sword,
+		/obj/item/tool/sword/saber,
+		/obj/item/tool/sword/katana,
+		/obj/item/tool/sword/katana/nano,
+		/obj/item/tool/sword,
+		/obj/item/tool/sword/improvised)
```

Comment:
I believe you can cut this all to just
`/obj/item/tool/sword`
I don't know how to do multi-line suggestions
But this is really cool, actually

Reply:
Yea just did in the new commit, thanks for spotting that